be/src/exec/parquet/file_reader.cpp

```diff
@@ -274,7 +272,8 @@ int FileReader::_get_partition_column_idx(const std::string& col_name) const {
     return -1;
 }
 
-Status FileReader::_decode_min_max_column(const tparquet::ColumnMetaData& column_meta,
+Status FileReader::_decode_min_max_column(const ParquetField& field, const std::string& timezone,
```

Comment:
You can simplify this code by using `PlainDecoder<PhysicalTypeTraits<column_meta.type>::CppType>`